packages/foundation/src/responsive/viewport.tsx

```diff
@@ -4,7 +4,7 @@ import { Breakpoints } from '../types';
 import { useWindowDimensions } from '../hooks/use-window-dimensions';
 import { BreakpointsSizes } from './breakpoints-sizes';
 
-type BreakpointString = 'l|s' | 'l|m' | 'm|s';
```

Comment:
A few changes here, as it didn't make a lot of sense to have all kind of combinations